src/components/Feed.jsx

```diff
@@ -0,0 +1,67 @@
+import moment from "moment"
+import PropTypes from "prop-types"
+import { useState } from "react"
+import "./Feed.css"
+
+export const Feed = ({ thoughts, fetchHappyThoughts }) => {
+  const [likes, setLikes] = useState({})
+  const [likedThoughts, setLikedThoughts] = useState([])
+
+  const onLikeIncrease = (_id) => {
+    // Send a request to the API to increase the hearts for the post with that ID
+    fetch(
+      `https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts/${_id}/like`,
+      { method: "POST" }
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        //Update the local state to reflect the new like count
+        setLikes((prevLikes) => ({
+          ...prevLikes,
+          [_id]: data.hearts,
+        }))
+        setLikedThoughts((prevLikedThoughts) => [...prevLikedThoughts, _id])
+        fetchHappyThoughts()
```

Comment:
I think this fetch call is not really needed as you update a particular post on which you set the likes, other than that, nothing else could really be changed by the user. This would cause unnecessary re-render on the whole App parent component.